tests/Feature/Collections/CollectionsTest.php

```diff
@@ -120,8 +123,8 @@
 it('Using Cassandra nested collections', function () use ($nestedCollections) {
     $session = scyllaDbConnection($nestedCollections);
 
-    $address = new Map(Cassandra\Type::text(), Cassandra\Type::text());
-    $addresses = new Map(Cassandra\Type::text(), $address->type());
+    $address = new Map(\Cassandra\Type::text(), \Cassandra\Type::text());
```

Comment:
Why are you instantiating a new object in the test? Couldn't you use a Mockery to mock these dependencies?

Reply:
No problem, thanks for the review

Reply:
Don't be sorry ever, good reviews are meant for challenging the author of the pull request, every question, suggestion or pointing the things that you missed are helpful, they provide valuable information to us to improve the readability of the code.